examples/issue/components/IssueEditor.js

```diff
@@ -36,7 +36,7 @@ export default class IssueEditor extends React.Component {
     };
   }
 
-  onChange = (editorState, cb) => this.setState({ editorState }, cb);
+  onChange = (editorState) => this.setState({ editorState });
```

Comment:
This is a React feature, not a draft-js. `setState` takes in an optional second argument callback